Formula/defaultbrowser.rb

```diff
@@ -0,0 +1,17 @@
+class Defaultbrowser < Formula
+  desc "Command-line tool for getting & setting the default browser"
+  homepage "https://github.com/kerma/defaultbrowser"
+  url "https://github.com/kerma/defaultbrowser/archive/1.1.tar.gz"
+  sha256 "56249f05da912bbe828153d775dc4f497f5a8b453210c2788d6a439418ac2ea3"
+
+  def install
+    system "make"
```

Comment:
This should be redundant now too. I just neglected to update upstream's `README` when I filed the prior PR.

Reply:
@DomT4 Oh, ok. Updated accordingly.